From akpm@osdl.org Mon Nov 21 22:55:00 2005
Message-Id: <200511220652.jAM6q32h026458@shell0.pdx.osdl.net>
From: "Jordan, William P" <William.Jordan@unisys.com>
Subject: PCI Hotplug: ibmphp_pci.c copy-n-paste fix
To: William.Jordan@unisys.com, greg@kroah.com
From: akpm@osdl.org
Date: Mon, 21 Nov 2005 22:51:57 -0800


From: "Jordan, William P" <William.Jordan@unisys.com>

I noticed what appears to be a cut/paste error in
drivers/pci/hotplug/ibmphp_pci.c:

Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/pci/hotplug/ibmphp_pci.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/pci/hotplug/ibmphp_pci.c
+++ gregkh-2.6/drivers/pci/hotplug/ibmphp_pci.c
@@ -969,7 +969,7 @@ static int configure_bridge (struct pci_
 			debug ("io 32\n");
 			need_io_upper = TRUE;
 		}
-		if ((io_base & PCI_PREF_RANGE_TYPE_MASK) == PCI_PREF_RANGE_TYPE_64) {
+		if ((pfmem_base & PCI_PREF_RANGE_TYPE_MASK) == PCI_PREF_RANGE_TYPE_64) {
 			debug ("pfmem 64\n");
 			need_pfmem_upper = TRUE;
 		}
